File: 코딩 기초 트레이닝/이어 붙인 수-p181928/이어 붙인 수.py
# https://school.programmers.co.kr/learn/courses/30/lessons/181928

def solution(num_list):
    answer = 0
    odd = ''
    even = ''

    for n in num_list:
        if n % 2 == 0:
            even += str(n)
        else:
            odd += str(n)

    return int(even) + int(odd)

File: 코딩 기초 트레이닝/이어 붙인 수-p181928/이어 붙인 수.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181928

function solution(num_list) {
    var answer = 0;
    let odd = '';
    let even = '';

    num_list.forEach((n) => {
        if(n % 2 === 0) {
            even += n
        } else {
            odd += n
        }
    })


    return Number(even) + Number(odd);
}
